### DISCUSSION TITLE
How to format Vue files?

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [LazyVim/LazyVim#877] [Q&A]
### Did you check the docs?

- [X] I have read all the LazyVim docs

### Is your feature request related to a problem? Please describe.

Unable to format in vue

### Describe the solution you'd like

i don’t know how to do 

### Describe alternatives you've considered

 I don’t know how to do

**Comment by hirotaka** [comment]
I use prettierd to format my Vue files.
https://www.lazyvim.org/plugins/extras/formatting.prettier
